frontend/src/components/navbar/NavbarSearchPanel.vue

```diff
@@ -56,6 +62,9 @@ export default {
                     page: 1
                 }
             });
+            setTimeout(() => {
```

Comment:
no, this data should be in state. After search icon clicked, vuex action executed and is_loading mutation become true, so preloaded will react automatically on state changes